helm: preflight chart lint failed

I ran `helm lint --with-subcharts --values values.yaml` in install/kubernetes/cilium directory and got the below error.

```shell scripts
$ helm lint --with-subcharts --values values.yaml
...
==> Linting charts/preflight
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/: template: preflight/templates/daemonset.yaml:78:7: executing "preflight/templates/daemonset.yaml" at <ne .Values.tofqdnsPreCache "">: error calling ne: invalid type for comparison

Error: 10 chart(s) linted, 1 chart(s) failed
```

--- a/.github/workflows/helm-check.yaml
+++ b/.github/workflows/helm-check.yaml
@@ -37,18 +37,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
-      - name: Run chart-testing (lint)
-        id: lint
-        uses: helm/chart-testing-action@v1.0.0-rc.2
-        with:
-          command: lint
-          config: install/kubernetes/chart-testing.yaml
+      - name: Run helm lint
+        run: |
+          cd install/kubernetes/cilium
+          helm lint --with-subcharts --values values.yaml
 
       - name: Create kind cluster
         uses: helm/kind-action@v1.0.0-rc.1
         with:
           install_local_path_provisioner: true
-        if: steps.lint.outputs.changed == 'true'
 
       # This test is just to make sure the helm chart can be installed. It will not replace existing integration tests
       - name: Run chart-testing (install)

--- a/install/kubernetes/cilium/charts/preflight/values.yaml
+++ b/install/kubernetes/cilium/charts/preflight/values.yaml
@@ -1,1 +1,13 @@
 image: cilium
+
+# Include the PreFlight DaemonSet
+enabled: false
+
+# Path to write the --tofqdns-pre-cache file to. When empty no file is
+# generated.
+tofqdnsPreCache: ""
+
+# By default we should always validate the installed CNPs before upgrading
+# Cilium. This will make sure the user will have the policies deployed in the
+# cluster with the right schema.
+validateCNPs: true

